sdk/cosmos/azure-cosmos/CHANGELOG.md

```diff
@@ -1,7 +1,16 @@
 ## Release History
 
-## 4.12.0-beta.1 (Unreleased)
+### 4.12.0 (2021-02-09)
+#### New Features
+* Added connection endpoint rediscovery feature to help reduce and spread-out high latency spikes.
```

Comment:
CX might not understand our internal names.

How about "Reducing high tail latencies through faster detection of stale peers"

Reply:
@kirankumarkolli  sorry Kiran, I missed your comments before I merge the PR.

For this one, since we have public API for setConnectionEndpointRediscoveryEnabled, so I think it might be good to mention it?